api/utils/dependencies.py

```diff
@@ -52,8 +52,12 @@ def get_super_admin(db: Session = Depends(get_db), token: str = Depends(oauth2_s
     if not user.is_super_admin:
         logger.error("User is not a super admin")
         raise HTTPException(
-            status_code=status.HTTP_403_FORBIDDEN,
-            detail="You do not have permission to access this resource",
+            status_code=401,
+            detail={
```

Comment:
Replace this dictionary with what was there before. The exception handler takes care of everything